app/routes.py

```diff
@@ -1 +0,0 @@
-from flask import Blueprint
```

Comment:
Nice call creating two different routes files for our respective models -- it keeps both`goal_routes.py` and `task_routes.py` clean and short!